### ISSUE TITLE
Merge a fork?

### ISSUE THREAD

**Issue by jakubsuchy** [MiniCodeMonkey/amazon-alexa-php#1]
The fork by @kora88 currently has all the necessary validation tools, could it be merged here so users don't discover an incomplete toolkit via Composer?

**Comment by MiniCodeMonkey** [comment]
It would be great to merge that!

**Comment by danstreeter** [comment]
Agreed - there are a lot of forked upstream fixes that fix some vital things. I'm using this library for a custom skill i'm working on - so might fork and merge all fixes in as one hit to get me going.

**Comment by emerham** [comment]
Looking at the forks and history of each one, @froodley https://github.com/froodley/amazon-alexa-php/network has done a lot of work with code.
*Edit*
I see now that froodley has identified it is a breaking change from the initial repo and it will no longer work as a possible merge, but there are some good things to possibly add back in. I was trying to get slots working with an app and see that some of the code they added to the Intent class helps in getting some of the data from the request.
